doc/man/man3/suite_create.3

```diff
@@ -8,7 +8,7 @@
 .\" Back-Cover Texts.  A copy of the license is included in
 .\" the section entitled "GNU Free Documentation License".
 .\"
-.TH "suite_create" "3" "October 25, 2020" "Check" "LibCheck Programmer's Manual"
+.TH "suite_create" "3" "October 25, 2020" "0.15.2" "LibCheck"
```

Comment:
I'm thinking that this will get out of date the next time there is a release. Is there a way to populate the value in the build, so it is not forgotten next release? Or, in the intention that the docs will only be updated when done so manually, and that the man pages only represent specifically the 0.15.2 release?

Reply:
I can make this a build dependency to automatically use the Autoconf/CMake version number, that's a great idea.

Reply:
The man pages now automatically use the version number defined in `configure.ac`. This is also true when using CMake instead of Autotools. In addition, the date is automatically set using a timestamp, and it is formatted as `YYYY-MM-DD`, as per the man page guidelines.